appveyor.yml

```diff
@@ -0,0 +1,30 @@
+image: ubuntu
+
+version: 3.6.0-{build}
+
+branches:
+  only:
+    - master
+
+max_jobs: 1
+
+dotnet_csproj:
+  patch: true
+  file: '**\*.csproj'
+  version: '{version}'
+  package_version: '{version}'
+  assembly_version: '{version}'
+  file_version: '{version}'
+  informational_version: '{version}'
+
```

Comment:
Need a `dotnet test` thing?

Reply:
It's picked up by convention

Reply:
Here's my last build output https://ci.appveyor.com/project/mr-sandy/carter/build/3.6.0-8

Reply:
interesting :)

Reply:
:)